spec/features/admin/non_admin_user_cannot_access_index_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+feature "Non admin user cannot access index" do
+  scenario "successfully" do
+    user = create(:user)
+    log_in_as user
+
+    visit admin_users_path
+
+    expect(page).to have_text(I18n.t("admin.defaults.require_admin"))
```

Comment:
Aren't we already testing this in a controller spec?

Reply:
Duh, of course - Done in sha: 33f2dfc